helpers.go

```diff
@@ -139,3 +141,29 @@ func decodeSlice(elemType reflect.Type, slice reflect.Value, input map[string]in
 
 	return nil
 }
+
+// timeHookFunc supports decoding to time
+func timeHookFunc(from reflect.Type, to reflect.Type, data interface{}) (interface{}, error) {
+
+	// Decode time.Time
+	if to == reflect.TypeOf(time.Time{}) {
```

Comment:
reflect.TypeOf(time.Time{}) is invariant so can be global e.g.
```go
var timeType = reflect.TypeOf(time.Time{})
```
This then becomes:
```
if to == timeType {
...
}
```